plugins/tasks/ansible/src/main/java/com/walmartlabs/concord/plugins/ansible/KerberosAuth.java

```diff
@@ -66,8 +67,14 @@ public KerberosAuth(String username, String password, Path tmpDir, boolean debug
     }
 
     @Override
-    public KerberosAuth enrich(AnsibleEnv env) {
-        env.put("KRB5CCNAME", tgtCacheFile.toString());
+    public KerberosAuth enrich(AnsibleEnv env, AnsibleContext context) {
+        String dockerImage = getString(context.args(), TaskParams.DOCKER_IMAGE_KEY.getKey());
+        if (dockerImage != null) {
+            env.put("KRB5CCNAME", tgtCacheFile.toString().replace(context.workDir().toString(), "/workspace"));
```

Comment:
It might be better to do something like
```java
Paths.get("/workspace").resolve(tgtCacheFile.relativize(workDir))
```
Instead of replacing all `/workspaces` in the path.

Reply:
Can we relativize two uncommon root paths? Because i got `KRB5CCNAME=/workspace/../../..,` when i ran above one

Reply:
But they should be common. The original `tgtCacheFile` value should start with `workDir`. It's not the case?

Reply:
It should be reverse `Paths.get("/workspace").resolve(context.workDir().relativize(tgtCacheFile))`.

Reply:
Right, I always forget the order in `relativize`.